android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/ProxiedForConsentedUsersApi.java

```diff
@@ -31,6 +31,7 @@
 /**
  * Proxies ForConsentedUserApi calls so it uses retrieves authenticated client from AuthenticationManager
  * to make the call.
+ * TODO: push refreshing of credentials into rest-client's authentication handler
  */
 class ProxiedForConsentedUsersApi implements ForConsentedUsersApi {
```

Comment:
I don't really understand the need for this class.  I thought this was handled already by the rest-client.  It seems very odd to have to reimplement all the methods this way.  Can you provide a little more explanation?